[2023-11-08T02:09:05.089+0000] {logging_mixin.py:151} INFO - Changing /opt/***/logs/dag_id=check_S3_minio_bucket_v02/run_id=manual__2023-11-08T02:08:32.907291+00:00/task_id=2nd_task permission to 509
[2023-11-08T02:09:05.144+0000] {logging_mixin.py:151} INFO - Changing /opt/***/logs/dag_id=check_S3_minio_bucket_v02/run_id=manual__2023-11-08T02:08:32.907291+00:00/task_id=2nd_task permission to 509
[2023-11-08T02:09:05.194+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: check_S3_minio_bucket_v02.2nd_task manual__2023-11-08T02:08:32.907291+00:00 [queued]>
[2023-11-08T02:09:05.218+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: check_S3_minio_bucket_v02.2nd_task manual__2023-11-08T02:08:32.907291+00:00 [queued]>
[2023-11-08T02:09:05.220+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 6
[2023-11-08T02:09:05.253+0000] {taskinstance.py:1382} INFO - Executing <Task(BashOperator): 2nd_task> on 2023-11-08 02:08:32.907291+00:00
[2023-11-08T02:09:05.266+0000] {standard_task_runner.py:57} INFO - Started process 43274 to run task
[2023-11-08T02:09:05.274+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'check_S3_minio_bucket_v02', '2nd_task', 'manual__2023-11-08T02:08:32.907291+00:00', '--job-id', '444', '--raw', '--subdir', 'DAGS_FOLDER/check_S3_bucket.py', '--cfg-path', '/tmp/tmpulo6sf9l']
[2023-11-08T02:09:05.292+0000] {standard_task_runner.py:85} INFO - Job 444: Subtask 2nd_task
[2023-11-08T02:09:05.387+0000] {logging_mixin.py:151} INFO - Changing /opt/***/logs/dag_id=check_S3_minio_bucket_v02/run_id=manual__2023-11-08T02:08:32.907291+00:00/task_id=2nd_task permission to 509
[2023-11-08T02:09:05.395+0000] {task_command.py:416} INFO - Running <TaskInstance: check_S3_minio_bucket_v02.2nd_task manual__2023-11-08T02:08:32.907291+00:00 [running]> on host 326a1fe25163
[2023-11-08T02:09:05.508+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='Toema' AIRFLOW_CTX_DAG_ID='check_S3_minio_bucket_v02' AIRFLOW_CTX_TASK_ID='2nd_task' AIRFLOW_CTX_EXECUTION_DATE='2023-11-08T02:08:32.907291+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2023-11-08T02:08:32.907291+00:00'
[2023-11-08T02:09:05.511+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2023-11-08T02:09:05.514+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', 'pip install urllib3']
[2023-11-08T02:09:05.534+0000] {subprocess.py:86} INFO - Output:
[2023-11-08T02:09:09.471+0000] {subprocess.py:93} INFO - Defaulting to user installation because normal site-packages is not writeable
[2023-11-08T02:09:10.545+0000] {subprocess.py:93} INFO - Requirement already satisfied: urllib3 in /home/***/.local/lib/python3.8/site-packages (1.26.17)
[2023-11-08T02:09:24.168+0000] {subprocess.py:93} INFO - 
[2023-11-08T02:09:24.170+0000] {subprocess.py:93} INFO - [notice] A new release of pip is available: 23.0.1 -> 23.3.1
[2023-11-08T02:09:24.172+0000] {subprocess.py:93} INFO - [notice] To update, run: pip install --upgrade pip
[2023-11-08T02:09:24.968+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2023-11-08T02:09:25.069+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=check_S3_minio_bucket_v02, task_id=2nd_task, execution_date=20231108T020832, start_date=20231108T020905, end_date=20231108T020925
[2023-11-08T02:09:25.244+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-11-08T02:09:25.355+0000] {taskinstance.py:2778} INFO - 1 downstream tasks scheduled from follow-on schedule check
